cardigan_new/stories/components/Buttons/Basics/Button/Button.stories.tsx

```diff
@@ -0,0 +1,123 @@
+import { Meta, StoryObj } from '@storybook/react';
```

Comment:
I think this file is one level too deep, could be `cardigan_new/stories/components/Buttons/Basics/Button.stories.tsx`

Reply:
Yes, I think you're probably right. I think if we stick with the naming of the stories being the same as the containing folder, then it should be `cardigan_new/stories/components/Buttons/Button/Button.stories.tsx` (but the `title` prop can still reference `Basics`)